### PR TITLE
handle groupv2 groups

### PR DESCRIPTION
**Pull request by ghost**
Hi everyone,
I've added three helper functions to handle legacy groups and groupv2 the same way. It works for me, but I would be glad if others could test and review.

The other commits are optional, I can also put them into separate PRs if that's desired:
`c5bc8b0` fixes the `set_log_level` function and
`89469d2` adds printed filenames of received attachements

### CODE REVIEW

**Comment by rxcomm** [comment]
I gave this a try. No problems that I see. I ran it with signald commit 81729030ec9b6b1fc94cc4d24a21b02894520980

**Comment by tych0** [comment]
Cool, I haven't tried it myself, but if it works for someone else it probably works. I'll try to check it out soon, thanks for fixing this!

**Comment by Mic92** [comment]
Also works for me. It only does not show invitations properly. But this is a separate feature.

File: signal.py [hunk 1]
```diff
@@ -212,7 +247,10 @@ def render_message(message):
     attachments = message.get('attachments')
     if attachments is not None:
         types = [attach['contentType'] for attach in attachments]
-        attachment_msg = "<sent {}> ".format(', '.join(types))
+        filenames = [attach['storedFilename'] for attach in attachments]
+        attachment_msg = "<sent {}>: \n{}".format(
+                ', '.join(types),
+                '\n'.join(filenames))
```
Comment (Mic92) [inline signal.py:253]: Currently there is no space between messages and attachments:

```python
        attachment_msg = "<sent {}>: \n{}\n".format(
                ', '.join(types),
                '\n'.join(filenames))
```

Comment (Mic92) [reply signal.py:253]: Example:

```
Shannan Lekwati │ <sent image/jpeg>:
                │ /home/joerg/.config/signald/attachments/arV_0-3Y5u9gdh-SVBXKFirst one’s in!
```

**Comment by ghost** [comment]
@Mic92 you're right. Sorry, I forgot to insert another line break in the formatted string. This will be fixed in #34

**Pull request merged**